boards/PDM/Src/Io/Io_Efuse.c

```diff
@@ -0,0 +1,231 @@
+#include <assert.h>
+#include <math.h>
+#include "Io_Efuse.h"
+#include "configs/Io_EfuseConfig.h"
+
+#pragma GCC diagnostic ignored "-Wconversion"
+
+struct Efuse_Context
+{
+    // The SPI handle for the SPI device the E-Fuse is connected to
```

Comment:
Mentioned this in a comment in Io_Efuses.c but gonna reiterate here again just to make sure. Lets put the CS port and pin in here so when we call the functions below, we just pass the struct ptr.

Reply:
DONE